srcpkgs/amule/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'amule'
+pkgname=amule
+version=11066
+revision=1
+wrksrc=amule-master
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man
+	--enable-cas --enable-wxcas --enable-amule-daemon
+	--enable-amulecmd --enable-amule-gui --enable-alc
+	--enable-alcc --enable-webserver --disable-debug
+	--enable-optimize --enable-geoip --disable-upnp
+	--enable-fileview --with-boost  --with-wx-config=/usr/bin/wx-config-gtk3"
+hostmakedepends="tar autoconf automake gettext gettext-devel wxWidgets-gtk3"
+makedepends="boost-devel zlib-devel wxWidgets-gtk3-devel gd-devel geoip-devel crypto++-devel"
+depends="wxWidgets-gtk3-devel gd-devel geoip-devel crypto++-devel"
```

Comment:
Why does this depend on `-devel` packages, xbps-src will usually look at the resulting binaries and automatically add dependencies by looking at linked shared libraries.
If a dependency is not automatically detected you need to add them, but generally not the -devel ones.

https://github.com/void-linux/void-packages/pull/26999/checks?check_run_id=1514556177#step:10:1156

Reply:
Fixed. The `makedepends` phase still require `*-devel` for the `configure` phase, which checks the presence of these libraries, to succeed.

Reply:
Fixed